packages/scaffold-ui/src/partials/w3m-connect-walletconnect-widget/index.ts

```diff
@@ -32,7 +38,8 @@ export class W3mConnectWalletConnectWidget extends LitElement {
       return null
     }
 
-    const connector = this.connectors.find(c => c.type === 'WALLET_CONNECT')
+    const connector = this.connectors.find(c => c.name === 'WalletConnect')
```

Comment:
Why is this filter changed?

Reply:
Oh it's bc we are not have `MULTI_CHAIN` types. This will handle the multi-chain version, but since it'll be unified with UPA, we could revert this.